src/components/weather/card.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import moment from "moment";
+import { toCelcius, toFahrenheit } from "utils/tempConverter";
+
+function Card({ data, tempPresenter }) {
+  const dayTemp =
+    tempPresenter === "celsius"
+      ? toCelcius(data?.temp?.day)
+      : toFahrenheit(data?.temp?.day);
+  const mornTemp =
+    tempPresenter === "celsius"
+      ? toCelcius(data?.temp?.morn)
+      : toFahrenheit(data?.temp?.morn);
+  return (
+    <div className="card-banner">
+      <div className="card-date">
+        {moment(data.dt * 1000).format("ddd, MMM Do")}
+      </div>
+      <div className="card-weather">
+        <div>
+          <img
+            src={`https://openweathermap.org/img/wn/${data?.weather?.[0]?.icon}.png`}
```

Comment:
What happens here if ```data``` or ```weather```  is undefined or null? Is there another way to make this more defensive to errors?

Reply:
And to add to that question, will this component render if ```data``` is null or undefined?